### PR TITLE
[venstarthermostat] PR related fixes

### PR DESCRIPTION
**Pull request by digitaldan**
fixes representation property key in discovery, adds thing documentation, removes some blank lines, adds author to CODEOWNERS, runs spotless formatter.

Signed-off-by: Dan Cunningham <dan@digitaldan.com>

### CODE REVIEW

**Comment by digitaldan** [comment]
see #7054

File: bundles/org.openhab.binding.venstarthermostat/README.md [hunk 1]
```diff
@@ -42,6 +42,12 @@ After adding the Inbox item, enter the user name and password from the physical
 | humidity           | Number             | Humidity                     |                                                        |
 
 
+### Thing Configuration
```
Comment (kaikreuzer) [inline bundles/org.openhab.binding.venstarthermostat/README.md:45]: The "Thing Configuration" section should have a table (similar to the channels above), which lists and describes the different parameters. It should NOT include the textual configuration example (which usually is provided in a "Full Example" section). I see that the whole README does not really follow our template for the docs...

**Review by kaikreuzer — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.venstarthermostat/README.md (hunk 1 above)
Comment (digitaldan) [reply bundles/org.openhab.binding.venstarthermostat/README.md:45]: Agreed, i'll clean up the README

File: bundles/org.openhab.binding.venstarthermostat/README.md [hunk 2]
```diff
@@ -16,17 +15,31 @@ without HTTPS and authentication, the binding doesn't support it, in an effort
 to provide as secure an installation as possible.
 
 When you've set the username and password, make a note of these, as you'll need
-to enter them in the thermostat configuration in OpenHAB.
+to enter them in the thermostat configuration in OpenHAB.
```
Comment (kaikreuzer) [inline bundles/org.openhab.binding.venstarthermostat/README.md:18]:
```suggestion
to enter them in the thermostat configuration in openHAB.   
```

File: bundles/org.openhab.binding.venstarthermostat/README.md [hunk 3]
```diff
@@ -16,17 +15,31 @@ without HTTPS and authentication, the binding doesn't support it, in an effort
...
 
 Once the binding is installed it will attempt to auto discovery Venstar thermostats located on the local network.
 These will appear as Things in the system Inbox. 
 After adding the Inbox item, enter the user name and password from the physical thermostat in the Thing's configuration.
 
+## Thing Configuration
+
+### ColorTouch Thermostat
+
+| Parameter | Description                                                                                                            |
```
Comment (kaikreuzer) [inline bundles/org.openhab.binding.venstarthermostat/README.md:37]: Could you add a column on whether it is mandatory or optional?

Comment (kaikreuzer) [reply bundles/org.openhab.binding.venstarthermostat/README.md:37]: (and in case of optional, what default is used?)

**Review by kaikreuzer — CHANGES_REQUESTED** [review]
Sorry, I am still complaining ;-)

**Comment by digitaldan** [comment]
While creating the documentation i also realized its not possible to create a thing manually, fortunately this is a small change, which i will address here.

**Comment by kaikreuzer** [comment]
As I'd like to get the spotless fixes into the repo, please let me merge this and ask you for another PR for the further changes - thanks!

**Comment by digitaldan** [comment]
Thanks @kaikreuzer , sorry about the fire drill!  See #8498

**Pull request merged**